### DISCUSSION TITLE
How can I attach to a zellij session on starting up Kitty.

### DISCUSSION THREAD

**Discussion by MoltenMonster** [kovidgoyal/kitty#7384] [Q&A]
[**Solved**](https://www.reddit.com/r/KittyTerminal/comments/1cbrgjw/comment/l113t4t/?utm_source=share&utm_medium=web2x&context=3)**:** I wrote in `launch.conf`: `launch zellij attach -c`.

I want `zellij attach -c` to run once Kitty is loaded up. I tried adding `startup_session launch.conf` to my config file, and writing `kitty @ send-text "zellij attach -c"` in launch.conf as well as `launch sh -c "zellij attach -c"` but neither works.

**Comment by Android789515** [comment]
Please share where launch.conf goes. I have no idea. I assume it's not for Kitty as adding `launch zellij attach -c` does nothing for me.

**Reply by Android789515** [reply]
Nvm I figured it out. For anyone else having this issue, you have to also set your `startup_session` in `kitty.conf` to `launch.conf`

kitty.conf
```
# ... other stuff

startup_session launch.conf

# ... other stuff
```

launch.conf
```
launch zellij attach -c
```
